docker-entrypoint.sh

```diff
@@ -329,12 +332,47 @@ liquibaseChangelockReset() {
   echo "End releasing liquibase changeloglock."
 }
 
+validatePostgresConnection() {
+  # https://stackoverflow.com/a/58784528/6921621
```

Comment:
A brief comment about why this is relevant would help me here personally.

Reply:
The alert container can start up before the database container.  If we can't connect to the database don't start alert because the alert container needs to read and write data to/from the database.

Reply:
@psantos1113 I think he means a comment within the file.